scripts/test-eunomia.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -eu
```

Comment:
Would it be ok to change this to `set -euo pipefail`?

Reply:
yes we can use, I will change that .